common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -74,6 +75,18 @@ class DB(ctx: JdbcContext[PostgresDialect, SnakeCase]) {
     }
   }
 
+  def insertImages(images: List[Image]): Unit = {
+    try {
+      val action = quote {
+        liftQuery(images).foreach(i => query[Image].insert(i))
+      }
+      ctx.run(action)
+    } catch {
+      case e: java.sql.BatchUpdateException => throw e.getNextException
```

Comment:
This is a one off process that is run, not sure of best logging practice? At the moment it goes as far as `println`s.